tasks.py

```diff
@@ -0,0 +1,185 @@
+                'DocumentConversion/ConvertToOffice/',
+                'DocumentConversion/CreateDocFromXPS/',
+                'DocumentConversion/FacturXConverter/',
+                'DocumentConversion/PDFAConverter/',
+                'DocumentConversion/PDFXConverter/',
+                'DocumentConversion/ZUGFeRDConverter/',
+                'DocumentOptimization/PDFOptimize/',
+                'Forms/ConvertXFAToAcroForms/',
+                'Forms/ExportFormsData/',
+                'Forms/FlattenForms/',
+                'Forms/ImportFormsData/',
+                'Images/DocToImages/',
+                'Images/DrawSeparations/',
+                'Images/EPSSeparations/',
+                'Images/GetSeparatedImages/',
+                'Images/ImageDisplayByteArray',
+                'Images/ImageEmbedICCProfile/',
+                'Images/ImageExport/',
+                'Images/ImageExtraction/',
+                'Images/ImageFromBufferedImage/',
+                'Images/ImageImport/',
+                'Images/ImageResampling/',
+                'Images/OutputPreview/',
+                'Images/RasterizePage/',
+                'InformationExtraction/ListBookmarks/',
+                'InformationExtraction/ListFonts/',
+                'InformationExtraction/ListInfo/',
+                'InformationExtraction/ListLayers/',
+                'InformationExtraction/ListPaths/',
+                'InformationExtraction/Metadata/',
+                'OpticalCharacterRecognition/AddTextToDocument/',
+                'OpticalCharacterRecognition/AddTextToImage/',
+                'Other/MemoryFileSystem/',
+                'Other/StreamIO/',
+                'Security/AddRegexRedaction/',
+                'Security/Redactions/',
+                'Text/AddGlyphs/',
+                'Text/AddUnicodeText/',
+                'Text/AddVerticalText/',
+                'Text/ListWords/',
+                'Text/RegexExtractText/',
+                'Text/RegexTextSearch/',
+                'Text/TextExtract/'
+        ]
+
+@task()
+def clean_samples(ctx):
+    """Cleans files that were generated from building the samples"""
+    for sample in samples_list:
+        full_path = os.path.join(os.getcwd(), sample)
+        with ctx.cd(full_path):
+            ctx.run('mvn clean')
+            ctx.run('git clean -fdx')
+
+@task()
+def build_samples(ctx):
+    """Builds the APDFL Java Maven samples"""
+    ctx.run('invoke clean-samples')
+    for sample in samples_list:
+        full_path = os.path.join(os.getcwd(), sample)
+
+        if platform.system() == 'Darwin' and ('ConvertToOffice' in sample or 'CreateDocFromXPS' in sample or 'ConvertXFAToAcroForms' in sample or 'ExportFormsData' in sample or 'FlattenForms' in sample or 'ImportFormsData' in sample):
+            output = "not available on this OS" if ('ConvertToOffice' in sample or 'CreateDocFromXPS' in sample) else "cannot be run in CI on this OS"
```

Comment:
Did `flake8` approve this line? It seems awfully long...

Reply:
Hmm, flake8 wasn't running for me locally, so I manually ran it and fixed other flake8 errors as well.